src/backend/launcher.ts

```diff
@@ -737,6 +737,13 @@ async function callRunner(
         rej('Process terminated with signal ' + signal)
       }
 
+      if (options?.runnerStdoutLog) {
+        const f = options.runnerStdoutLog
+        appendFileSync(f, `[${new Date().toLocaleString()}] ${safeCommand}\n`)
```

Comment:
It might be a good idea to do this before any command runs...

Reply:
done